### ISSUE TITLE
Goerli multicall address

### ISSUE THREAD

**Issue by c0rv0s** [cavanmflynn/ethers-multicall#5]
deployed at address 0x77dca2c955b15e9de4dbbcf1246b4b85b651e50e, chainID 5. no reason not to add this in

**Comment by c0rv0s** [comment]
included in this pr: https://github.com/cavanmflynn/ethers-multicall/pull/6

**Comment by cavanmflynn** [comment]
Released in [v0.2.0](https://github.com/cavanmflynn/ethers-multicall/releases/tag/v0.2.0)

**Comment by Nisthar** [comment]
> Released in [v0.2.0](https://github.com/cavanmflynn/ethers-multicall/releases/tag/v0.2.0)

hey man is this library still maintained?

I tried with a simple contract and its not working for me.
```js
 const ethcallProvider = new Provider(provider, 56)
    await ethcallProvider.init()
    const contractCalls = []
    const swapContract = new Contract(swapAddress, swapABI)
swapContract .swap(); //undefined

    ```
```
TypeError: swapContract .swap is not a function
```
